src/embed.ts

```diff
@@ -2,21 +2,12 @@ import all from 'it-all';
 import * as result from './result';
 import makeRequest, { RequestError } from './request';
 
-/**
- * Available models for text embedding
- * @public
- */
-export type EmbedModel = 'textembedding-gecko';
```

Comment:
Let's keep this to help people autocomplete model names and detect errors. This is also allows to change the backing endpoint as needed (i.e. v1beta only has to support this model).